_includes/2018-hero.html

```diff
@@ -9,16 +9,16 @@ <h3 style="margin-bottom: 1.3em; margin-top: 0;">Toronto, Ontario</h3>
   <!-- Event hero section -->
   <section>
     <div class="feature">
-      <a class="splash-link" href="/yami-ichi/"><h4 class="splash-heading">Kickoff Yami-ichi {% include graphics/bullet.svg %} 13</h4></a>
-      <p>Network field trips and Toronto's first internet yami-ichi</p>
+      <a class="splash-link" href="/yami-ichi/"><h4 class="splash-heading">Internet Yami-ichi {% include graphics/bullet.svg %} 13</h4></a>
+      <p>Kickoff with network field trip and Toronto's first Yami-ichi</p>
```

Comment:
`Toronto's` -> `Toronto’s`

Reply:
I'd been relying on markdown rendering for these, but oops stuck in html tags! Fixed